update_reminder.sh

```diff
@@ -1,10 +1,36 @@
 #!/bin/bash
 MARKT_LAST_CHANGE=$(wget -q -O- https://wo-ist-markt.de/cities/berlin.json | sed -n 's/.*aktualisiert am \(.*\)",/\1/p')
-BERLIN_LAST_CHANGE=$(wget -q -O- https://daten.berlin.de/datensaetze/berliner-wochen-und-tr%C3%B6delm%C3%A4rkte | sed -n '/.*Aktualisiert.*/,$p' | sed -e '/.*end.*/,$d' | sed -n 's/.*<span .*>\(.*\)<\/span.*/\1/p')
+BERLIN_HEADER_AND_SITE=$(wget --server-response -q -O- https://daten.berlin.de/datensaetze/berliner-wochen-und-tr%C3%B6delm%C3%A4rkte  2>&1  )
 
+#Check Response is 200 (fix https://github.com/wo-ist-markt/wo-ist-markt-berlin-update-reminder/issues/7 )
+BERLIN_STATUS=$(echo "$BERLIN_HEADER_AND_SITE" | sed -n 's|.*HTTP/... \([0-9]*\).*|\1|p' )
+if [[ "$BERLIN_STATUS" != "200" ]]; then
+  echo "daten.berlin.de: returned a non 200-code"
+  echo "$BERLIN_STATUS"
+  exit 1
+fi
+
+#Check if GeoJSON API exists (fix https://github.com/wo-ist-markt/wo-ist-markt-berlin-update-reminder/issues/8)
+BERLIN_GEOJSON_URL=$(echo "$BERLIN_HEADER_AND_SITE" | sed -n 's/.*\(https.*all.geojson\).*/\1/p' )
+if [[ "$BERLIN_GEOJSON_URL" == "" ]]; then
+  echo "daten.berlin.de: GeoJSON-URL not available!"
+  exit 1
+fi
+
+#Check GeoJSON Response is 200 (fix https://github.com/wo-ist-markt/wo-ist-markt-berlin-update-reminder/issues/9 )
+BERLIN_GEOJSON_STATUS=$(wget --server-response "$BERLIN_GEOJSON_URL" 2>&1 | sed -n 's|.*HTTP/... \([0-9]*\).*|\1|p'  )
+if [[ "$BERLIN_GEOJSON_STATUS" != "200" ]]; then
+  echo "daten.berlin.de: GeoJSON-API returned a non 200-code"
+  echo "$BERLIN_GEOJSON_STATUS"
+  exit 1
+fi
+
+#Check if update date of data changed
```

Comment:
Maybe each check could be function with a nice descriptive function name. This would make comments needless. Also variables could be short because their scope is smaller. (not sure if that is actually true for shell scripts)